.gitignore

```diff
@@ -27,7 +27,7 @@ MANIFEST
 
 # PyInstaller
 #  Usually these files are written by a Python script from a template
-#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+#  before PyInstaller builds the exe file, so as to inject date/other information into it.
```

Comment:
Please fix this in upstream https://github.com/github/gitignore/blob/master/Python.gitignore
```suggestion
#  before PyInstaller builds the exe, so as to inject date/other info into it.
```